src/App/Layout.tsx

```diff
@@ -1,34 +1,16 @@
 import { Suspense } from "react";
 import { Outlet } from "react-router-dom";
-import { Link } from "./types";
 import Loader from "components/Loader";
 import Seo from "components/Seo";
-import { AP_LOGO, BASE_DOMAIN } from "constants/common";
-import { appRoutes } from "constants/routes";
+import { AP_LOGO } from "constants/common";
+import { IS_AST } from "constants/env";
 import Footer from "./Footer";
 import Header from "./Header";
-import { GROUPS_DATA, SOCIAL_MEDIA_LINKS } from "./constants";
+import { AST_LINKS, NON_AST_LINKS } from "./constants";
 
-const HEADER_LINKS: Link[] = [
-  { title: "For Non-Profits", href: BASE_DOMAIN, external: true },
-  {
-    title: "Marketplace",
-    href: appRoutes.marketplace,
-  },
-  {
-    title: "Giving Partners",
-    href: `${BASE_DOMAIN}/giving-partners-csr/`,
-    external: true,
-  },
-  {
-    title: "About",
-    href: `${BASE_DOMAIN}/about-angel-giving/`,
-    external: true,
-  },
-  { title: "Register", href: appRoutes.register },
-  // NOTE: governance will be reenabled when we relaunch the $HALO token
-  // { title: "Governance", href: appRoutes.govern },
-];
+const { HEADER_LINKS, GROUPS_DATA, SOCIAL_MEDIA_LINKS } = IS_AST
+  ? AST_LINKS
+  : NON_AST_LINKS;
```

Comment:
Nit: Can we call this `CHARITY_LINKS` since it is for Charity Endowments to be more clear?